runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java

```diff
@@ -88,14 +89,26 @@ public Dimension render(Graphics2D graphics, Point parent)
 				position.setLocation(tooltip.getPosition());
 			}
 
-			if (lastLocation.contains(position))
+			// check if this tooltip would overlap the last
```

Comment:
This looks like it is fixing a bug with the tooltip renderer? It should be in a separate commit with a proper commit message

Reply:
Done